Jenkinsfile

```diff
@@ -388,8 +388,18 @@ pipeline {
         always {
           junit testResults: '**/target/surefire-reports/*.xml'
           container('maven') {
-            // clean up the redis namespace
-            sh "kubectl delete namespace ${NAMESPACE_REDIS} --ignore-not-found=true"
+            script {
+              try {
+                 // clean up the redis namespace
+                // workaround for Cannot delete namespace still in Terminating #NXP-3228
+                sh """
+                  kubectl get namespace ${NAMESPACE_REDIS} -o json | sed -e s/\\"kubernetes\\"//g  | kubectl replace --raw /api/v1/namespaces/${NAMESPACE_REDIS}/finalize -f -
+                  kubectl delete namespace ${NAMESPACE_REDIS} --ignore-not-found=true --cascade=true --force=1 --grace-period=0
```

Comment:
I think we can remove `--cascade=true --force=1 --grace-period=0` since a plain deletion is expected to work here.

Reply:
thanks,
 we can delete this options as you said